eng/native/configurecompiler.cmake

```diff
@@ -117,27 +117,27 @@ if (MSVC)
     # Debug build specific flags
     # The Ninja generator doesn't appear to have the default `/INCREMENTAL:ON` that
     # the Visual Studio generator has. Therefore we will override the default for Visual Studio only.
-    add_link_options(/INCREMENTAL:NO DEBUG)
-    add_link_options(/OPT:REF DEBUG)
-    add_link_options(/OPT:NOICF DEBUG)
+    add_link_options($<$<CONFIG:DEBUG>:LINKER:/INCREMENTAL:NO>)
```

Comment:
We should only need the `LINKER:` prefix for arguments that were passed with `-Wl,` beforehand. Were you seeing errors without it for the other arguments?